src/sentry/filters/legacy_browsers.py

```diff
@@ -11,14 +14,53 @@
     'Safari': 6,
     'Edge': 0,
     'Opera': 15,
+    'Android': 4,
 }
 
 
+class LegacyBrowserFilterSerializer(serializers.Serializer):
+    # TODO: maybe find a serializer that handles multple values already
+    active = serializers.BooleanField()
+    subfilters = MultipleChoiceField(choices=[
+        'ie8',
+        'ie9',
+        'opera',
```

Comment:
I'd +1 this. I don't think we're intending to 100% block all of Opera, Safari, and Android. Considering modern Opera and Safari iirc use literally the same engines. No idea what state of Android land is, but seems heavy handed to discard them all.

Even if the code isn't intending to do this, it'd make sense to make this more granular in case we need to add more toggles in the future. That way we aren't stuck with `opera` as a key value and need to manage that forever in the database. If it's even something like, `opera_pre_5` or whatever, that makes it easier to add more without dealing with old/awkward data.